packages/LUIS/bin/luis.js

```diff
@@ -265,7 +265,7 @@ async function runProgram() {
                                 return;
                             }
                         }
-                        result = await client.apps.deleteMethod(args.region, args.cloud, args.appId, args.force, args);
+                        result = await client.apps.deleteMethod(args.region, args.cloud, args.appId, { force: args.force }, args);
```

Comment:
Code change should have a corresponding test change.

Reply:
This was not working at all. Can you advise on where to find the original test to fix it?

Reply:
It wasn't working because there was no test. :)